README.md

```diff
@@ -12,24 +12,46 @@ assumption that your host doesn't have port 69 in use.
 Usage
 -----
 
+`init.sh` assumes that netboot directory is `./netboot` or provide it as a
+variable `NETBOOT`.
+
+
 ```
-git clone https://github.com/3mdeb/pxe-server.git
-./init.sh
+git clone https://github.com/miczyg1/pxe-server.git
+git checkout diskless-netboot
+./init.sh [NETBOOT=/path/to/netboot/dir]
 ```
 
 APU2 development and testing
 ----------------------------
 
 ## Prepare iPXE
 
-For those who want to test apu2 I advise to use [3mdeb/netboot](https://github.com/3mdeb/netboot) configuration.
-After finished `init.sh` as described above. Run:
+You have to provide PXE bootloader from any debian-like system netboot
+installation.
+
+Copy the `menu.cfg` and `syslinux.cfg` in correct directory. Do not forget to
+change nfsroot path and nfsroot server ip.
```

Comment:
@pietrushnic I have not yet commited to `3mdeb/netboot` so the changes I made to menu configs are not present there. I just created this PR quickly for You to use it as soon as possible